src/molecules/Hero/Hero.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment, useState } from "react";
+
+import {
+  FullscreenContainer,
+  HeroContainer,
+  IllustrationWrapper,
+  Legend,
+  PanelHolder,
+  StyledDialog,
+  StyledPanel,
+  StyledTransition,
+  Title,
+} from "./Hero.styled";
+import { ReactComponent as HeroImage } from "assets/akol_beseder.svg";
+import HowToUse from "molecules/HowToUse/HowToUse";
+import Button from "atoms/Button/Button";
+import { Dialog, Transition } from "@headlessui/react";
+
+const Hero = () => {
+  const [isPopupOpen, setIsPopupOpen] = useState(false);
+  return (
+    <HeroContainer>
+      <IllustrationWrapper>
+        <HeroImage />
+      </IllustrationWrapper>
+      <Title>Automation roadmap </Title>
+      <Legend>From zero to automation hero</Legend>
+      <Button onClick={() => setIsPopupOpen(true)}>How to use</Button>
+      <Transition appear show={isPopupOpen} as={Fragment}>
+        <Dialog as={StyledDialog} onClose={() => setIsPopupOpen(false)}>
+          <Transition.Child
+            as={StyledTransition}
+            enter="enter"
+            enterFrom="enterFrom"
+            enterTo="enterTo"
+            leave="leave"
+            leaveFrom="leaveFrom"
+            leaveTo="leaveTo"
+          />
+          <PanelHolder>
+            <FullscreenContainer>
+              <Dialog.Panel as={StyledPanel}>
+                <Dialog.Title as="h3">How to use this roadmap?</Dialog.Title>
+                <HowToUse />
+                <Button onClick={() => setIsPopupOpen(false)}>Close</Button>
```

Comment:
when reusing same functions it's good to have them explicitly declared in the component block for better reusability and optimisation 🙂